.yamato/test_versions.metafile

```diff
@@ -0,0 +1,6 @@
+# List of editor versions for standalone-build-test and its dependencies.
```

Comment:
This basically gets "included" at the start of each of the test files, so that they're all using the same versions. See https://internaldocs.hq.unity3d.com/yamato/yaml/templating/#include-metadata-files for more details